src/components/DateRange/DateRange.scss

```diff
@@ -1,11 +1,18 @@
 .date-range {
     &__input {
         min-width: 190px;
+        height: 28px;
         padding: 5px 8px;
 
         color: var(--g-color-text-primary);
         border: 1px solid var(--g-color-line-generic);
         border-radius: var(--g-border-radius-m);
+        outline: none;
         background: transparent;
     }
+
+    &__input:focus,
+    &__input:focus-visible {
+        border: 1px solid var(--g-color-line-generic-hover);
+    }
```

Comment:
These changes fix light outline, which doesn't correspond to dark mode and makes control height more, than others

Before:
![Screen Shot 2024-05-15 at 17 09 31](https://github.com/ydb-platform/ydb-embedded-ui/assets/67755036/d4186bf8-36ad-4bc9-9c18-6c39132c7eb6)

After:
![Screen Shot 2024-05-15 at 17 11 56](https://github.com/ydb-platform/ydb-embedded-ui/assets/67755036/ce9b9763-5844-4f0b-aa5c-6656fca68f3b)